roles/tendrl-storage-node/tasks/tendrl-node-agent.yml

```diff
@@ -44,3 +45,5 @@
   service:
     name=tendrl-node-agent
     state=started
+  notify:
```

Comment:
Yes its required. We can have any number of notify but only one will be executed at the end.
The install node agent return no state change most of the time. If package not found or package version is old then also it does not provide a state change because the update package happens at top level for all the packages. So we need this here!. However in some cased we need for install node agent also.